lib/ansible/modules/monitoring/zabbix/zabbix_template.py

```diff
@@ -605,6 +605,10 @@ def import_template(self, template_content, template_type='json'):
             if LooseVersion(api_version).version[:2] <= LooseVersion('3.2').version:
                 update_rules['applications']['updateExisting'] = True
 
+            if (LooseVersion(api_version).version[:3] >= LooseVersion('4.0.16').version) or \
```

Comment:
```suggestion
            if (LooseVersion(api_version).version[:3] == LooseVersion('4.0.16').version) or \
```

Please change the conditional expression because this problem will not occur in the following version.

* 4.0.15
* 4.4.0
* 4.4.1
* 4.4.2
* 4.4.3